Assets/Scripts/UI/InGameUI/DevConsole/DevConsole.cs

```diff
@@ -815,7 +818,9 @@ private void LoadCommands()
                 new Command("GetRoomIDs", CoreCommands.GetAllRoomIDs, "Logs all the room IDs"),
                 new Command<int, string, Vector3>("DoBuild", CoreCommands.DoBuild, CoreCommands.DoBuildHelp),
                 new Command("DirtyTileGraph", CoreCommands.InvalidateTileGraph, "Invalidates the tile graph"),
-                new Command("GetCharNames", CoreCommands.GetCharacterNames, "Logs all the character names"));
+                new Command("GetCharNames", CoreCommands.GetCharacterNames, "Logs all the character names"),
+                new Command<Vector2>("NewCharacter", CoreCommands.NewCharacter, "Creates a new character at position provided"),
```

Comment:
Firstly, yes they should however they are using some older code to get the current tile (which I haven't updated to be 3D), I'm going to update it in my DevConsole2.0 Revisions PR thingy so should we just leave it till then?  The next point is odd?? Neither of them have any code that would do that functionality, the center just calls the new character with a center position?  Also I'm looking to remove center position code to replace it with defaults (which are already in place in PR #1680).  And yeh same with the third.  I'm thinking that due to the small nature of this either I just close this PR and incorporate changes in my bigger revision PR since I'm moving the location of all commands in that too (to make them more available to modders), or I guess we can just merge it since while it may not have optimum it has pretty close (though personally I would rather closing it down and just doing the changes in my revision PR due to reasons stated)?